apps/events/models.py

```diff
@@ -114,6 +114,10 @@ class Event(models.Model):
     """Event type. Used mainly for filtering"""
     organizer = models.ForeignKey(Group, verbose_name=_('arrangør'), blank=True, null=True, on_delete=SET_NULL)
     """Committee responsible for organizing the event"""
+    visible = models.BooleanField(
+        _('Vis arrangementet utenfor Dashboard/admin-panelet'),
+        default=True,
+        help_text='Denne brukes primært for å skjule eksisterende eventer.')
```

Comment:
I think wording it as 'arrangementer' would be more correct than 'eventer'

Reply:
I agree, fixed